tests/modules/catalog/test_glossary.py

```diff
@@ -1,8 +1,7 @@
 from datetime import datetime
 
-from dataall.core.catalog.db.glossary_models import GlossaryNode
-from dataall.core.environment.db.models import Environment
-from dataall.core.organizations.db.organization_models import Organization
+from dataall.modules.catalog.db.glossary_models import GlossaryNode
+from tests.api.conftest import *
```

Comment:
Don't we need the `Environment` and `Organization` classes in line 9 for example?

Reply:
They are being taken from `from tests.api.conftest import *` which is bad indeed. However, no worries:  #656 comes to rescue 😄